Proteomics/tutorials/WIP_proteinQuant_SIL.md

```diff
@@ -20,16 +27,14 @@ This tutorial deals with protein quantitation via stable isotope labelling (SIL)
-
-> ### :nut_and_bolt: Comment: OpenMS MS1 Quantitation Tools
-> The OpenMS suite features quite a lot of different MS1 feature detection tools (called "FeatureFinder"s). Momentarily (as of February 2017), the recommended standard tool is the **FeatureFinderCentroided** :wrench:. It has been thoroughly tested, but is not longer improved and may be replaced by the newer **FeatureFinderMultiplex** :wrench: in future. We will focus on the first tool here, but you may well try the later one if it more suits your need. A recent development is the **FeatureFinderIdentification** :wrench:. Unlike the other FeatureFinders in the OpenMS suite, it takes PSM IDs as an input. However, it is still under development and not recommended as default tool right now (as of February 2017). For more information on OpenMS FeatureFinders, please consult [this discussion](https://github.com/OpenMS/OpenMS/issues/2424#issuecomment-282293381). 
+Quantitation on MS1 level may in principle be carried out without prior knowledge of peptide / protein IDs. However, some quantitation algorithms take the IDs as an input to make sure that every PSM that was identified is also quantified. This is not the case in our example here. 
+In the OpenMS suite, most of the provided tools for MS1 feature detection quantify solely based on mzML files. The advantage of this approach is that quantitations can be made on strict criteria to reduce misquantitations. The drawback is that not all IDs can be matched to a quantitation later on in the workflow. 
+The tool settings need to be carefully tested and evaluated manually to obtain optimal results. We will explain this in the section [Evaluation and Optimization of Quantitation Results](#evaluation-and-optimization-of-quantitation-results). 
 
 > ### :pencil2: Hands-on: MS1 Feature Detection
 > 
-> Here, we will use a quantitative comparison of **HEK _OR_ E.coli** cell lysate as a test dataset. In this experiment, the very same cell lysate was once labelled with light, once with heavy **dimethyl _OR_ SILAC** and both samples were subsequently mixed in a certain ratio. Your objective in this hands-on-tutorial is to find out the correct mixing ratio. *And no, the ratio is not mentioned in the description of the dataset on PRIDE. We left out that particular information on purpose. Haha.*
-> 
-> 
+> Here, we will use a quantitative comparison of **HEK _OR_ E.coli** cell lysate as a test dataset. In this experiment, the very same cell lysate was once labelled with light, once with heavy **dimethyl _OR_ SILAC** and both samples were subsequently mixed in a certain ratio.
```

Comment:
And I liked this part so much...

But can I keep the sentence about the objective? "Your objective in this hands-on-tutorial is to find out the correct mixing ratio."